modules/serverless-rn-1-10-0.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-10-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.10.0
+
+[id="new-features-1-10-0_{context}"]
+== New features
+* {ServerlessProductName} now uses Knative Operator 0.16.0.
+* {ServerlessProductName} now uses Knative Serving 0.16.0.
+* {ServerlessProductName} uses Knative Eventing 0.16.0.
+* {ServerlessProductName} now uses Kourier 0.16.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.16.1.
+* The annotation `knative-eventing-injection=enabled` that was previously used to annotate namespaces for broker creation is now deprecated. The new annotation is `eventing.knative.dev/injection=enable`. For more information, see the documentation on _Creating a broker as a cluster administrator using namespace annotation_.
```

Comment:
I'm curious if the "New Features" heading should be right above this line. Or the list of versions is considered as "new features"? ...If I look at the rendered document we are not consistent with this across releases.

Reply:
So the list always goes under new features, it's supposed to be consistent for all releases but I see that it's not in 1.9.0, so let me fix that one.